web3deployment/reserveDeployer.js

```diff
@@ -153,9 +236,9 @@ function parseInput( jsonInput ) {
 
       tokens.push(address);
       const dict = {
-        minimalRecordResolution : web3.utils.toBN(val["minimalRecordResolution"]),
-        maxPerBlockImbalance : web3.utils.toBN(val["maxPerBlockImbalance"]),
-        maxTotalImbalance : web3.utils.toBN(val["maxTotalImbalance"])
+        minimalRecordResolution : val["minimalRecordResolution"],
+        maxPerBlockImbalance : val["maxPerBlockImbalance"],
+        maxTotalImbalance : val["maxTotalImbalance"]
       };
```

Comment:
I don't follow, why don't we need BN here anymore?
or it was never needed?

Reply:
The web3.utils.toBn function returns a BN object. Passing this into the function will also fail and return an error like `<rejected> Error: invalid number value (arg="minimalRecordResolution", coderType="uint256", value="38d7ea4c68000")`

The values stored in the config file (ropsten.json) is already a string, there is no need to convert it to BN.

Reply:
As for the package-lock, I wasn't sure why it changed when I did npm-install. Anyway, I will remove it.

Reply:
yes please remove it.
we have started using npm ci for a new repo clone
and not npm i.

anyway package lock shouldn't be added to PRs unless packages changed as well.

Reply:
Understood. Thanks for letting me know!